doc/ug/dv_methodology/index.md

```diff
@@ -299,60 +299,112 @@ Once the regression tool is developed, it will provide a way to enable these cap
 
 ## Coverage Collection
 
-Collecting, analyzing and reporting coverage with waivers is another requirement to assert 'verification complete'.
+Collecting, analyzing, and reporting coverage with waivers is a requirement to assert 'verification complet'.
 Any gaps in our measured coverage need to be understood and either waived (no need to cover) or closed by additional testing.
 The end goal is to achieve 100% coverage across all applicable coverage metrics.
-This process known as "coverage closure", is done in close collaboration with the designer(s).
+This process is known as "coverage closure", and is done in close collaboration with the designer(s).
 Coverage collected from all tests run as a part of the regression is merged into a database for analysis.
-Our primary tool of choice for our coverage closure needs is Synopsys' VCS & Verdi.
-However, the use of other tools / simulators are welcome.
+Our primary tool of choice for our coverage closure needs is Synopsys VCS & Verdi.
+However, the use of other simulators is welcome.
+
+Why do we need coverage?
+
+The common answer is to flush out bugs in the design.
+This is not accurate enough. Making sure there are no bugs in a design, While it is important to make sure there are no bugs in a design, this is not sufficient.
+One must also make sure the design works as intended.
+That is, it must provide all the functionality specified in the design specification.
+So a more precise answer for why we need coverage is to flush out flaws in the design.
+These flaws can be either design bugs or deficiencies in the design with respect to the specification.
```

Comment:
```suggestion
So, we need coverage to flush out not only the bugs in the design but also deficiencies in the specification. 
```

Reply:
@sriyerg  I think I prefer my version.
but if you insist I don't feel that strongly about it.?

Reply:
I suppose Sri's suggestion has the advantage that it doesn't call a deficiency in the specification a flaw in the design.

Reply:
If we can convey the same idea in a more concise way, I'd prefer that. I will leave this upto you. I am fine either way.